docs/src/man/crun-vm.md

```diff
@@ -0,0 +1,65 @@
+crun-vm 1 "User Commands"
+==================================================
+
+# NAME
+
+crun-vm - Run VMs as easily as you run containers
+
+# SYNOPSIS
+
+**crun-vm** \[OPTIONS\] \<COMMAND\>
+
+# DESCRIPTION
+
+crun-vm is an OCI Runtime that enables Podman, Docker, and Kubernetes to run QEMU-compatible Virtual Machine (VM) images.
+
+# COMMANDS
+
+  create      Create a container
```

Comment:
Can you move this into a table, it  will look a ton better. And high light the command.